src/sentry/web/frontend/auth_login.py

```diff
@@ -150,7 +156,7 @@ def handle_basic_auth(self, request: Request, **kwargs):
             # HACK: grab whatever the first backend is and assume it works
             user.backend = settings.AUTHENTICATION_BACKENDS[0]
 
-            login(request, user, organization_id=organization.id if organization else None)
+            self._handle_login(request, user, organization)
```

Comment:
@corps Is determining the active org after a login attempt a pre-existing behaviour?

Reply:
Yes, unfortunately, as I'm finding.  the `get_active_organization` calls fails until after the `request.user` is set, even in the original code paths.  There many more `get_active_organization` calls than there are `determine_active_organization` now, and there is some side effects that play off each other.

Reply:
Actually, I am partially wrong -- yes, in the case of the `get_active_organization` call bellow, but not in the case of the invite flow (that uses the original organization).  This logic is a little hard to get my head around, but there are some tests that cover this and I got a slightly modified version coming.